Firestore/src/Query.php

```diff
@@ -321,94 +323,34 @@ public function select(array $fieldPaths)
      * $query = $query->where('friends', 'array-contains', ['Steve', 'Sarah']);
      * ```
      *
-     * @param string|FieldPath $fieldPath The field to filter by.
+     * ```
+     * use Google\Cloud\Firestore\Filter;
+     *
+     * // Filtering with Filter::or
+     * $query = $query->where(Filter::or([Filter::field('firstName', '=', 'John'),
+     *                          Filter::field('firstName', '=', 'Monica')]));
```

Comment:
Make this more readable:
```suggestion
     * $query = $query->where(Filter::or([
     *     Filter::field('firstName', '=', 'John'),
     *     Filter::field('firstName', '=', 'Monica')
     * ]));
```

Reply:
looks good, thank you! done

Reply:
Pointed out other places where this need to be fixed

Reply:
Thank you, I have changed the places you pointed, thank you!